mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -744,9 +745,9 @@ func newTFNotificationModelListV2(n []admin.AlertsNotificationRootForGroup, curr
 				Roles:          value.Roles,
 				ChannelName:    conversion.StringPtrNullIfEmpty(value.ChannelName),
 				DatadogRegion:  conversion.StringPtrNullIfEmpty(value.DatadogRegion),
-				DelayMin:       types.Int64Value(int64(*value.DelayMin)),
+				DelayMin:       types.Int64PointerValue(util.IntPtrToInt64Ptr(value.DelayMin)),
```

Comment:
I am not sure why you need `Int64PointerValue` and `util.IntPtrToInt64Ptr`.

Reply:
`IntervalMin` and `DelayMin` are of type *int, as in some case they may not be defined. `Int64PointerValue` and `util.IntPtrToInt64Ptr` are used to safely store the optional value into the model, without insecurely accessing the underlying value as done in `*value.DelayMin`.